chargeio/models/Credit.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using ChargeIo.Infrastructure;
+using ChargeIO;

```

Comment:
fixed in https://github.com/affinipay/chargeio-net/pull/6/commits/d30b734f6bce7f7710448d5c0885fe374dba93a1